servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/publisher/TimeoutPublisherTest.java

```diff
@@ -83,7 +83,7 @@ boolean restartAtOnNext() {
     }
 
     @RegisterExtension
-    final ExecutorExtension<TestExecutor> executorExtension = ExecutorExtension.withTestExecutor();
+    static final ExecutorExtension<TestExecutor> executorExtension = ExecutorExtension.withTestExecutor();
```

Comment:
Should this use `setClassLevel(true)` now?

Reply:
`ExecutorExtension` doesn't provide a `setClassLevel(true)` option but I will create a separate RFE PR.